README.md

```diff
@@ -1,31 +1,84 @@
+<p align="center"><a href="https://solidus.io/" target="_blank" rel="noopener noreferrer"><img width="250" src="./logo.svg" alt="Solidus logo"></a></p>
+
+<p align="center">
+  <a href="https://circleci.com/gh/solidusio/solidus/tree/master"><img src="https://circleci.com/gh/solidusio/solidus/tree/master.svg?style=shield" alt="Circle CI"></a>
+  <a href="https://rubygems.org/gems/solidus"><img src="https://img.shields.io/gem/v/solidus.svg" alt="Gem"></a>
+  <a href="http://img.shields.io/badge/license-BSD-yellowgreen.svg"><img src="http://img.shields.io/badge/license-BSD-yellowgreen.svg" alt="License"></a>
+  <a href="http://slack.solidus.io"><img src="http://slack.solidus.io/badge.svg" alt="Slack"></a>
+  <a href="https://opencollective.com/solidus"><img src="https://opencollective.com/solidus/backers/badge.svg" alt="Backers on Open Collective"></a>
+  <a href="https://opencollective.com/solidus"><img src="https://opencollective.com/solidus/sponsors/badge.svg" alt="Sponsors on Open Collective"></a>
+  <br>
+  <a href="https://www.codetriage.com/solidusio/solidus"><img src="https://www.codetriage.com/solidusio/solidus/badges/users.svg" alt="Open Source Helpers"></a>
+  <a href="https://houndci.com"><img src="https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg" alt="Reviewed by Hound"></a>
+</p>
+
+<p align="center">
+  <strong>A free, open-source ecommerce platform that gives you complete control over your store.</strong>
+</p>
```

Comment:
I am not a fan of using more and more html in our readme file. First it is a markdown file and secondary this file is also meant to be read in an editor or terminal. Please do not see the GitHub preview as the single way to view this file.